run_tests.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
```

Comment:
Do we need this? I've come to think our projects don't need a special `run_tests.py`, that running `pytest` directly is simpler, which is why I didn't include it in the cookiecutters. Any configuration/defaults we want to include could be put in the pytest configuration file.

Reply:
I probably copied this from other repos. We use pytest now, so I can help to clean up run_test.py in other repos too.